org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "http://schemas.opentranslatorstothings.org/org.opent2t.sample.hub.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T Hub",
+  "definitions": {
+    "org.opent2t.sample.hub.superpopular": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "ReadOnly, Gets the name of the hub."
+        },
+        "devices": {
+          "description": "ReadOnly, Gets the current set of devices associated with the hub."
+        }
```

Comment:
I'll look into the OCF bit.

On your other question, you're right it is a bit inefficient, the flow is the following: 

instantiate hub
get devices
using each item in the returned array, create a device translator
get details of each device

Now if you have a wink hub with 10 devices you'll end up with 11 calls to get the full details.  We need to support the alljoyn protocol and each of the translators need to run in that environment.  There we can't marshal objects back and forth only data, hence the need to instantiate each translator individually.

---
In reply to: [76336787](https://github.com/openT2T/translators/pull/75#discussion_r76336787) [](ancestors = 76336787,76335569,76334693,76154047)